extern/ttmath/ttmathtypes.h

```diff
@@ -205,7 +205,7 @@ namespace ttmath
 	/*!
 		on 64bit platforms one word (uint, sint) will be equal 64bits
 	*/
-	#ifdef _MSC_VER
+	#ifdef _WIN32
 		/* in VC 'long' type has 32 bits, __int64 is VC extension */
```

Comment:
I see no problems with check for  _WIN32 here - just want to make sure you've checked ttmath on mingw, initially it seems not to be in a list of supported platforms.

Reply:
__WIN32 is "more standard" and __WIN32__ is even more but IIRC it is not supported by MSVC so better to check for Firebird-specific WIN_NT.

Reply:
I have reverted this change..

Reply:
Isn't MSYS2 a package manager too?
Should not the firebird build depends on shared tom libraries?

Reply:
> Isn't MSYS2 a package manager too? 

Yes

> Should not the firebird build depends on shared tom libraries?

Yes, it is already depends on tom libs on MSYS2.